app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_create :update_comment_counter
```

Comment:
**OPTIONAL** -I suggest using the `after_save` method instead, as it ensures that the update is made only when the data is saved in the database. Please fix this :pray: